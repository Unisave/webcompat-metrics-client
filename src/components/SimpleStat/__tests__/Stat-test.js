/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import { Stat } from "..";

it("renders Stat default correctly", () => {
  const stat = {
    label: "test label",
    count: 3,
  };
  const tree = renderer
    .create(
      <Stat stat={stat}>
        <div>{"Stat"}</div>
      </Stat>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders SimpleStat default correctly with custom style", () => {
  const stat = {
    label: "test label",
    count: 3,
  };
  const tree = renderer
    .create(
      <Stat stat={stat} style={{ color: "blue" }}>
        <div>{"Stat"}</div>
      </Stat>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
