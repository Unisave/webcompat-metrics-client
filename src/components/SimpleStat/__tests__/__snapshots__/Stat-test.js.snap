// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SimpleStat default correctly with custom style 1`] = `
<div
  className="stat"
>
  test label : 3
</div>
`;

exports[`renders Stat default correctly 1`] = `
<div
  className="stat"
>
  test label : 3
</div>
`;
